src/plugins/intel_gpu/include/intel_gpu/graph/kernel_impl_params.hpp

```diff
@@ -131,6 +134,17 @@ struct kernel_impl_params {
         return std::static_pointer_cast<const PType>(desc)->type == PType::type_id();
     }
 
+    void set_input1_tensors(tensor& tensor) const {
+        input1 = tensor;
+    }
+
+    void set_input2_tensors(tensor& tensor) const {
+        input2 = tensor;
+    }
+
+    tensor get_input1_tensors() const { return input1; }
+    tensor get_input2_tensors() const { return input2; }
+
```

Comment:
1. At creation of impl, we only need the "values". Why should we save it as tensor? 
(e.g., we can store the dim sizes std::vector<size_t> which contains width, height. )
2. Also I think we can generalize such runtime data in future using BinaryOutputBuffer as a next step

Reply:
Got it! I modified it and binarybuffer will be implemented in next phase.